From: David Rosca <nowrep@gmail.com>
Date: Thu, 15 Oct 2015 14:21:29 +0000
Subject: digitalclock: Use Text.HorizontalFit for date label in vertical panel
X-Git-Url: http://quickgit.kde.org/?p=plasma-workspace.git&a=commitdiff&h=4f265eda41df4abc6aa7fb2a5e2a1e73ae2989e3
---
digitalclock: Use Text.HorizontalFit for date label in vertical panel

Otherwise the text eliding works really weird (eg. at some point date is
not elided and when you make the panel wider it gets elided).

BUG: 351768
FIXED-IN: 5.4.3
REVIEW: 125625
---


--- a/applets/digital-clock/package/contents/ui/DigitalClock.qml
+++ b/applets/digital-clock/package/contents/ui/DigitalClock.qml
@@ -224,6 +224,7 @@
                 height: timezoneLabel.height
                 width: timezoneLabel.width
 
+                fontSizeMode: Text.HorizontalFit
                 minimumPixelSize: theme.smallestFont.pixelSize
                 elide: Text.ElideRight
             }
